src/pages/Cores/LearnMore/index.tsx

```diff
@@ -0,0 +1,80 @@
+import {articlesData} from './ArticlesData';
+import {SFC} from 'types';
+import {useToggle} from 'hooks';
+import * as S from './Styles';
+import ArticleCard from './ArticleCard';
+import CoreModal from 'modals/CoreModal';
+import EmptyText from 'components/EmptyText';
+
+const LearnMore: SFC = ({className}) => {
+  const [coreModalIsOpen, toggleCoreModal] = useToggle(false);
+  const articles = articlesData;
+
+  const renderContent = () => {
+    if (!!articles.length) return renderArticles();
+    return <EmptyText>No articles to display.</EmptyText>;
+  };
+
+  const renderArticles = () => {
+    const coreCards = articles.map((article) => {
+      if (article.type === 'list') {
+        return <ArticleCard article={article} key={article.id} element={renderDataInArticles(article.data)} />;
+      } else if (article.type === 'normal') {
+        return <ArticleCard article={article} key={article.id} />;
+      } else if (article.type === 'code') {
+        return <RenderCode data={article} />;
+      }
+    });
+    return <S.CardsContainer>{coreCards}</S.CardsContainer>;
+  };
```

Comment:
Instead of rendering the cards like this.
We can render them separately.
This would give us more flexibility on how to show/manage all cards.

Reply:
I did rendered the cards separately, now!